mlir/test/Target/LLVMIR/llvmir.mlir

```diff
@@ -1496,50 +1496,53 @@ llvm.func @elements_constant_3d_array() -> !llvm.array<2 x array<2 x array<2 x i
 // CHECK-LABEL: @atomicrmw
 llvm.func @atomicrmw(
     %f32_ptr : !llvm.ptr, %f32 : f32,
+    %f16_vec_ptr : !llvm.ptr, %f16_vec : vector<2xf16>,
     %i32_ptr : !llvm.ptr, %i32 : i32) {
   // CHECK: atomicrmw fadd ptr %{{.*}}, float %{{.*}} monotonic
   %0 = llvm.atomicrmw fadd %f32_ptr, %f32 monotonic : !llvm.ptr, f32
```

Comment:
Also, minor nitpick, could you move this to be the last test? In this way you don't have to renumber all the SSAs